setup.py

```diff
@@ -15,7 +14,6 @@
 DEFINE_MACROS = None
 FLAG_COVERAGE = '--cython-coverage'  # custom flag enabling Cython line tracing
 BASEDIR = os.path.abspath(os.path.dirname(__file__))
```

Comment:
@trexfeathers We could opt to use `pathlib` here instead of `os`, but I'm not greatly fussed :neutral_face:

Reply:
dd3d78045f80b46fd41d05eed494bedb0651ecc7

Reply:
Above and beyond :rocket: